### PR TITLE
archive oudated governance

### LINKED ISSUE #523: It would be great that we can add English name in the maintainer/committer list
For maintainer: https://github.com/pingcap/community/blob/master/architecture/maintainer-list.md
For the committer, it will link to a page on tidb.io with Chinese content only. https://tidb.io/archived/events//organization/

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

closes #523
refer #525

### CODE REVIEW

File: archive/governance/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Thanks to TiDB Maintainers for maintaining the TiDB project in an early stage.
+
+| GitHub ID                               | Name       | Native Name | Company |
+| :-------------------------------------- | :--------- | :---------- | :------ |
+| [shenli](https://github.com/shenli)     | Li Shen    | 申砾        | PingCAP |
+| [jackysp](https://github.com/jackysp)   | Jack Yu    | 于帅鹏      | PingCAP |
+| [coocood](https://github.com/coocood)   | Evan Zhou  | 周昱行      | PingCAP |
+| [winkyao](https://github.com/winkyao)   | Wink Yao   | 姚维        | PingCAP |
+| [zz-jason](https://github.com/zz-jason) | Jian Zhang | 张建        | PingCAP |
```
Comment (tisonkun) [inline archive/governance/README.md:15]: @shenli @jackysp @coocood @winkyao @zz-jason please confirm if this is proper English name for you.

Comment (jackysp) [reply archive/governance/README.md:15]: OK for me.

File: archive/governance/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+This folder contains outdated governance contents. Thanks for the mechanisms and members actively worked ever, which leads to what TiDB is today.
+
+## TiDB Maintainers
+
+Thanks to TiDB Maintainers for maintaining the TiDB project in an early stage.
+
+| GitHub ID                               | Name       | Native Name | Company |
+| :-------------------------------------- | :--------- | :---------- | :------ |
+| [shenli](https://github.com/shenli)     | Li Shen    | 申砾        | PingCAP |
```
Comment (shenli) [inline archive/governance/README.md:11]: I would prefer "Li Shen (申砾)"

Comment (tisonkun) [reply archive/governance/README.md:11]: ... and keep one "name" column?

**Review by winkyao — APPROVED** [review]
LGTM

File: archive/governance/README.md (hunk 2 above)
Comment (tisonkun) [reply archive/governance/README.md]: updated. PTAL @shenli

**Review by sykp241095 — APPROVED** [review]
/LGTM

**Pull request merged**